.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
```

Comment:
- It's unclear why removing the '##' from `RELEASE_NOTES` is necessary. Make sure this change aligns with the overall goal of the script and document the reason for this modification for future reference.